k8s/README.md

```diff
@@ -0,0 +1,262 @@
+# Airflow on/with Kubernetes
+These instructions document how to deploy the Airflow Dockerfile in the
+root repo to a Kubernetes cluster. Airflow can then also use the same
+Kubernetes cluster to run work via the 
+[KubernetesPodOperator](https://airflow.readthedocs.io/en/stable/kubernetes.html#kubernetes-operator).
+
+**NB** This doesn't describe how to setup the 
+[KubernetesExecutor](https://airflow.readthedocs.io/en/stable/kubernetes.html#kubernetes-executor),
+which creates pods to execute other types of operators (e.g. Bash, Python).
```

Comment:
"Use of the KubernetesExecutor is **not** necessary to use airflow to schedule the orchestration of pods in a kubernetes cluster. You can use this README to deploy airflow to a kubernetes cluster and schedule jobs on it."

Reply:
sure